specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -215,8 +215,8 @@
         "Healthcare",
         "ExtractiveSummarization",
         "CustomEntityRecognition",
-        "CustomSingleClassification",
-        "CustomMultiClassification"
+        "CustomSingleClass",
```

Comment:
mmmmm, I'm not sure if we want to change this or not. I was thinking of the list name only, not the task name.

Reply:
that seems counter-intuitive!
you want to unify the naming across all services, but at the same time, same api has 2 different conventions for both request and responses?!

Reply:
The reason for the different names in request and responses has to do with the use of the discriminator pattern.  The discriminator actually needs different names here in order to differentiate the schemas.  My preference would have been to have the names the same, but without the difference the Swagger definition will not pass model validation.